LandSandBoat/server
# [SQL] Add Ice Brand and Flametongue item_mods.sql, and a couple other item_mod corrections

Also corrected a few mistakes in other items I found.



**_I affirm:_**

## What does this pull request do?
Adds in the item modifiers for Ice Brand and Flametongue. I also corrected a couple things with some other items I found had issues.

## Steps to test these changes
As GM, !additem 20672 and 20673. Make sure stats show properly, and hit things with them to make sure the additional effects work properly.
